cli/command/context/import.go

```diff
@@ -0,0 +1,69 @@
+package context
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/docker/cli/cli"
+	"github.com/docker/cli/cli/command"
+	"github.com/docker/cli/cli/context/store"
+	"github.com/spf13/cobra"
+)
+
+type importOptions struct {
+	use   bool
+	force bool
+	name  string
+}
+
+func newImportCommand(dockerCli command.Cli) *cobra.Command {
+	opts := importOptions{}
+	cmd := &cobra.Command{
+		Use:   "import [OPTIONS] FILE",
+		Short: "Import a context",
+		Args:  cli.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return runImport(dockerCli, opts, args[0])
+		},
+	}
+
+	flags := cmd.Flags()
+	flags.BoolVar(&opts.use, "use", false, "Use this context by default")
+	flags.BoolVar(&opts.force, "force", false, "Overwrite any existing context with the same name")
+	flags.StringVar(&opts.name, "name", "", "Name of the context")
```

Comment:
This should be a 2nd argument instead

Reply:
I made it first, for having it in the same order as export